### DISCUSSION TITLE
Is it possible to hide 'learn more and customize' button?

### DISCUSSION THREAD

**Discussion by Pawel2505** [dargmuesli/nuxt-cookie-control#90] [Ideas]
Is it possible to hide 'learn more and customize' button? It would be just fine to have only two buttons 'accept' and 'decline' without using modal in some situations. Is it possible to achieve it at the moment?

**Comment by dargmuesli** [comment]
It's not implemented because that would go against the main cookie policy laws (GDPR, ...), at least to my understanding. In fact, that would be a feature suggestion I'd deny a PR for as long as noone with proper legal background convinces me that that feature can be optional.
